core/src/mindustry/world/Tile.java

```diff
@@ -446,7 +446,7 @@ public Tile getNearby(int rotation){
         return null;
     }
 
-    public Building getNearbyEntity(int rotation){
+    public Building getNearbyBuilding(int rotation){
```

Comment:
I think just `adjacentBuild` would be better, `getNearbyBuilding` is a mouthful and adjacent describes it better.